packages/ui5-proxy-middleware/test/ui5/middleware.test.ts

```diff
@@ -0,0 +1,415 @@
+//@ts-ignore
```

Comment:
@Klaus-Keller I am getting an error because of missing types.
![image](https://user-images.githubusercontent.com/91968887/155288821-9c14c8a9-81b7-426d-a8e9-74bac80ef27c.png)

Reply:
@Klaus-Keller But I the `@ui5/project` only for this single unit test. Do I really need to add a declaration file?

Reply:
For test case it's fine like this